zerver/lib/narrow.py

```diff
@@ -97,20 +97,35 @@ def read_stop_words() -> List[str]:
     return stop_words_list
 
 
+# "stream" is a legacy alias for "channel"
+channel_operators: List[str] = ["channel", "stream"]
```

Comment:
Do we need to add something that banes passing both `channel` and `stream` in the same narrow construction with different values?

Reply:
So, at the moment if we fetch messages with a narrow with two different streams, it doesn't return an error. It just returns no matching messages. After these changes, it's the same for a narrow with two channel operators or a channel and a stream operator (with different operands).

Therefore, if want to ban doing that, we should probably do so as a prep commit and that might also possibly merit a changelog entry.

Reply:
OK, seems fine to take no action on this for now.